business/dashboards.go

```diff
@@ -190,20 +190,24 @@ func (in *DashboardsService) GetDashboard(params model.DashboardQuery, template
 	for i, item := range dashboard.Spec.Items {
 		go func(idx int, chart v1alpha1.MonitoringDashboardChart) {
 			defer wg.Done()
+			unitScale := 1.0
+			if chart.UnitScale != 0.0 {
```

Comment:
This is a little weird.  I would expect:

`if chart.UnitScale != 1.0 {`

Reply:
it's because 0.0 is the default float value so it's what we get when it's missing in YAML ... and I think we wouldn't want to force people to set a value (also for backward compatibility)